layouts/analytics.html

```diff
@@ -10,4 +10,13 @@
     disable_session_recording: true
   })
 </script>
+<% else %>
+<script>
+  // Dummy PostHog client
+  const posthog = {};
+  posthog.init = function () {};
+  posthog.identify = function (arg) { console.log(`PostHog identify: ${arg}`); };
+  posthog.capture = function (event, data) { console.log(`PostHog capture - Event: ${event} - Data: ${JSON.stringify(data)}`); };
+  window.posthog = posthog;
```

Comment:
```suggestion
  // Dummy PostHog client
  window.posthog = {
    init() {},
    identify(arg) { 
      console.log(`PostHog identify: ${arg}`); 
    },
    capture(event, data) { 
      console.log(`PostHog capture - Event: ${event}`, data);
    }
  }
```
personal preference, but would terser to define the dummy object at once, also IMO its more useful to see the data as an object _(second param passed to console.log)_ rather than a JSON string